app/controllers/api/base_controller.rb

```diff
@@ -0,0 +1,29 @@
+module Api
```

Comment:
@yunixon authorization methods (and this controller holds only authorization method) should be in v1 because they may change in future versions. And same with errors handling. For example in next version we may decide to use xml. Or SOAP or something else.